### ISSUE TITLE
ToStringTests fails

### ISSUE THREAD

**Issue by offa** [mattgodbolt/seasocks#31]
I have a test failure of *ToStringTests* with following error (Gcc 6):

```
[…]/seasocks/src/test/c/ToStringTests.cpp:32: FAILED:
  {Unknown expression after the reported line}
due to unexpected exception with message:
  locale::facet::_S_create_c_locale name not valid

===============================================================================
test cases:    45 |    44 passed | 1 failed
assertions: 65939 | 65938 passed | 1 failed

```

According to [this](http://stackoverflow.com/questions/3221022/why-does-stdlocale-locen-us-throw-an-exception), the locale name strings are implementation specific.

**Comment by mattgodbolt** [comment]
Thanks: I'll take a look and most likely remove this test if I can't find a portable solution.

**Comment by offa** [comment]
One way could be the usage of `std::to_string()` where possible (maybe everywhere?) and deprecation of `toString()`.

**Comment by offa** [comment]
Solved by #32
